Update total-rewards.md

iogungbade opened the pull request:
<!-- Thank you for contributing a pull request! Here are a few tips to help you:

1. Learn how to update the handbook: https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-update-handbook
2. If this is your first contribution, please sign the Contributor License Agreement: https://mattermost.com/mattermost-contributor-agreement/
   - Once signed, you will be added to the Mattermost Approved Contributor List: https://docs.google.com/spreadsheets/d/1NTCeG-iL_VS9bFqtmHSfwETo5f-8MQ7oMDE5IUYJi_Y/pubhtml?gid=0&single=true
   - If you included your mailing address, you may receive a Limited Edition Mattermost Mug as a thank you gift after your pull request is merged: https://forum.mattermost.com/t/limited-edition-mattermost-mugs/143
-->

#### Summary
<!--
A description of what this pull request does.
-->

iogungbade committed:
Update total-rewards.md

iogungbade commented:
I added a section on "how we determine compensation".

natalie-hub reviewed (changes requested):
I reviewed feedback/edits with @iogungbade directly, and she'll make changes to this PR.

iogungbade committed:
Update total-rewards.md

Updated the "how we determine compensation" section to reflect Natalie's feedback.

natalie-hub reviewed (dismissed):
Looks good, sorry for delay

cwarnermm commented on operations/workplace/people/total-rewards.md:
```diff
@@ -21,6 +21,48 @@ To learn more about how the total rewards guiding principles drive equitable and
+
+We pay competitive local market rates. This means that our compensation system targets pay at or around the median salaries of comparable organizations in each local market. The local market compensation approach is based on an extensive internal analysis of the cost of labour (local market compensation) in over 12,000 cities across the globe, and it enables Mattermost to support equitable standa ...
+
+- _**Why we align compenstion to the local market**_ 
```
```suggestion
- _**Why we align compensation to the local market**_ 
```

cwarnermm committed:
Update operations/workplace/people/total-rewards.md

justinegeffen committed:
Update total-rewards.md

justinegeffen commented:
/check-cla

justinegeffen commented:
/check-cla
